sw/airborne/modules/computer_vision/opticflow/opticflow_calculator.c

```diff
@@ -584,12 +650,10 @@ void calc_edgeflow_tot(struct opticflow_t *opticflow, struct opticflow_state_t *
   int16_t der_shift_y = 0;
 
   if (opticflow->derotation) {
-    der_shift_x = (int16_t)(edge_hist[current_frame_nr].rates.p  /
-                            result->fps *
-                            (float)img->w / (OPTICFLOW_FOV_W) * opticflow->derotation_correction_factor_x);
-    der_shift_y = (int16_t)(edge_hist[current_frame_nr].rates.q /
-                            result->fps *
-                            (float)img->h / (OPTICFLOW_FOV_H) * opticflow->derotation_correction_factor_y);
+    der_shift_x = (int16_t)(edge_hist[current_frame_nr].eulers.phi *
```

Comment:
This seems like an angle instead of a rate: shouldn't there be something like (phi1 - phi2)?